packages/merlin/merlin.3.2.2/opam

```diff
@@ -0,0 +1,76 @@
+opam-version: "2.0"
+name: "merlin"
+version: "3.1.0"
```

Comment:
This is wrong. This line should be removed

Reply:
Indeed. That'll teach me, what was I think relying on tools for that sort of things.